frontend/src/components/Graph/index.tsx

```diff
@@ -323,6 +326,51 @@ function Graph({
 		buildChart();
 	}, [buildChart, forceReRender]);
 
+	useEffect(() => {
```

Comment:
why useEffect here ? isref not working ?

Reply:
Okay